analysis/deep2_r23_o32/calibration_plots.py

```diff
@@ -115,8 +116,8 @@ def lac_gpc_plots(fitspath, fitspath_ini, dataset, revised=False, individual=Fal
         OH = [det_OH, rlimit_OH, der_OH, der_OH_MACT]
         c_var = ['b', 'g', 'r', 'm']
 
-    local_analog_calibration.main(lR23, lO32, OH, LAC_out_pdf, yra=[7.0, 9.0], ctype=c_var,
-                                    label=['Detection', 'Robust Limits', 'DEEP2', 'MACT'], silent=False, verbose=True)
+    # local_analog_calibration.main(lR23, lO32, OH, LAC_out_pdf, yra=[7.0, 9.0], ctype=c_var,
+                                    # label=['Detection', 'Robust Limits', 'DEEP2', 'MACT'], silent=False)
```

Comment:
```suggestion
    # local_analog_calibration.main(lR23, lO32, OH, LAC_out_pdf, yra=[7.0, 9.0], ctype=c_var,
    #                               label=['Detection', 'Robust Limits', 'DEEP2', 'MACT'], silent=False)
```

Reply:
no longer applies. This was me trying to fix an issue and I committed it without changing it back.